Extension/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "license": "SEE LICENSE IN LICENSE.txt",
   "engines": {
-    "vscode": "^1.53.0"
+    "vscode": "^1.57.0"
```

Comment:
Don't forget to increase the minimum required version for Insiders.  :)

Reply:
Done